client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -264,11 +265,16 @@ const getRawAddressErrors = ( appState, addressData, siteId, shouldValidatePhone
 			);
 		}
 	}
+	if ( true === destinationPhone && !phone ) {
+		errors.phone = translate(
```

Comment:
For custom purposes, in line 255, it would go validate the phone number to match a 10 digit phone number format in origin. Since WCS&T supports US store only. The 10 digit rule is fine. 

In destination, we don't do any phone number validation (ie. i can put a single digit and it would still be okay). But I don't know if we should add international phone number validation here with this PR. Maybe something to add in the future?

Reply:
I avoided phone number validation here because of the plethora of different phone formats and didn't want to potentially annoy someone with an imperfect regex validation. 🤷